recipe/meta.yaml

```diff
@@ -78,6 +78,7 @@ test:
     - conda debug --help
     # bdist_conda
     - python test_bdist_conda_setup.py bdist_conda --help
+    - python -m pip check
```

Comment:
Move this to the top of the commands list (otherwise the comments don't make sense)

Reply:
Also need to add pip to `test/requires`